OneMore/Properties/Resources.ar-SA.resx

```diff
@@ -3446,6 +3446,10 @@ ISO-code then comma then language name</comment>
     <value>بحث ونسخ / نقل</value>
     <comment>ribbon item</comment>
   </data>
+  <data name="ribNaviButton_Label" xml:space="preserve">
+    <value>FastNavi</value>
```

Comment:
Is there a better name than "FastNavi"?

Reply:
Will fix it. How about "Incremental Search"?